pkg/microservice/user/core/repository/orm/user.go

```diff
@@ -149,3 +149,10 @@ func UpdateUser(uid string, user *models.User, db *gorm.DB) error {
 	}
 	return nil
 }
+
+func CheckUserMysqlDbHealthz() error {
+	if err := core.DB.Model(&models.User{}).Select("1").Limit(1).Error; err != nil {
+		return err
+	}
+	return nil
+}
```

Comment:
it is not part of repo layer, move to pkg/microservice/user/core/service/healthz.go